ignite/distributed/comp_models/base.py

```diff
@@ -5,7 +5,7 @@
 import torch
 from packaging.version import Version
 
-_torch_version_le_112 = Version(torch.__version__) > Version("1.12.0")
+_torch_version_ge_112 = Version(torch.__version__) > Version("1.12.0")
```

Comment:
```suggestion
_torch_version_gt_112 = Version(torch.__version__) > Version("1.12.0")
```

Reply:
Touché! To be more precise, based on [this](https://pytorch.org/docs/stable/notes/mps.html) it looks like we want ge 1.12.3, shall I change it to that? Otherwise, yes, gt_112